coalib/parsing/FilterHelper.py

```diff
@@ -33,3 +33,20 @@ def get_filtered_bears(cls, filter, args, all_bears=None):
         if not args or len(args) == 0:
             return all_bears
         return cls.available_filters[filter](all_bears, args)
+
+    @classmethod
+    def filter_bears(cls, bears, args):
+        """
+        Returns bears after filtering based on ``args``. It returns
+        intersection of bears if more than one elements present in ``args``
+        list.
+        :param bears:   Bears
+        :param args:    List of args based on ``bears`` has to be filtered. For
+        example: [['language', 'c', 'java'], ['can_fix', 'syntax']]
+        :return:        Filtered bears
```

Comment:
I've tried to reduce space b/w param name and description. But `python setup.py docs` is giving error on that.